.github/CODEOWNERS

```diff
@@ -1015,6 +1021,18 @@
 # ServiceLabel: %Network - Mobile %Mgmt
 # ServiceOwners:                                                   @ArthurMa1978
 
+# PRLabel: %Network - DNS Resolver
+/sdk/dnsresolver/Azure.ResourceManager.*/                          @jamesvoongms @jotrivet
+
+# ServiceLabel: %Network - DNS Resolver
+# ServiceOwners:                                                   @jamesvoongms @jotrivet
+
+# PRLabel: %Network - Cloud
```

Comment:
Why are you adding these?  The label was renamed in `main` and this entry shouldn't exist.

Reply:
Fixed, thanks for the catch